CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+#### 3.7.0
```

Comment:
I see that the user agent still returns 3.6.4, which is a minor update. To me, it makes sense that it's a minor, as I don't think this will break any existing code using this library.
Can you simply change this in the changelog? Thanks!

Reply:
@sergicastellsague my bad, fixed!

I did intend for this to be 3.6.4 but must have slipped in the log